lib/yocaml/build.ml

```diff
@@ -106,7 +106,7 @@ let fold_dependencies arr_list =
   (fun task -> { dependencies; task }), tasks
 ;;
 
-let failable_task task = { dependencies = Deps.empty; task }
+let simple_task task = { dependencies = Deps.empty; task }
```

Comment:
I agree that "failable task" wasn't a great name (but that was mainly to be able to fail freely) but I'm not sure that "simple" is more appropriate. How about "effectful_task" or something more ... clear?

Reply:
I have change `task` and `dependencies` for `get_task` and `get_dependencies` and I'have change `simple_task` to `lift_task`. Is that Ok for you?

Reply:
Yes, it's a better name. Thanks.